src/components/button/CdrButton.vue

```diff
@@ -85,7 +85,6 @@ const withBackgroundClass = computed(() => props.iconOnly
                        iconOnlyClass,
                        iconLeftClass,
                        iconRightClass,
-                       elevatedClass,
```

Comment:
where did this one go?

Reply:
Just a forgotten argument removal from here https://github.com/rei/rei-cedar-next/commit/81430ade56d921dffca1f01e98c1823c20fa8842